### ISSUE TITLE
Allow more fine control over the hub connection

### ISSUE THREAD

**Issue by UmairB** [Omhet/redux-signalr#3]
It would be nice to allow more fine control over the hub connection, e.g. being able to call  `withAutomaticReconnect` on the hub connection builder.

**Comment by Omhet** [comment]
Hi! 
Yes, it's actually a great idea.

However, what do you think if I add a possibility to pass a built connection instead of passing all the parameters?
This way one would be able to build whichever connection she would like to.

**Comment by UmairB** [comment]
That would be much more flexible yes. Maybe something like

```
const signalR = signalMiddleware({
    callbacks,
    connectionBuilder => connectionBuilder.withUrl(...
});
```

Or maybe pass the connection object itself yes. And leave the middleware to bind the callbacks and invoke functions...

**Comment by Omhet** [comment]
I released 1.0.0 of this package right now.

Made a connection object passing obligatory. So, closing this issue.
Please, see updated README (step 1)
